helm/ph-ee-engine/connector-ams-mifos/templates/deployment.yaml

```diff
@@ -33,7 +33,7 @@ spec:
         image: "{{ .Values.image }}:{{ .Values.imageTag }}"
         imagePullPolicy: "{{ .Values.imagePullPolicy }}"
         ports:
-          - containerPort: 5000
+          - containerPort: 8080
```

Comment:
check the channel connector to be sure that it is not connecting ams directly with the port